presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduDistributedQueries.java

```diff
@@ -79,6 +79,7 @@ public void testDataMappingSmokeTest(DataMappingTestSetup dataMappingTestSetup)
     }
 
     @Test
+    @Deprecated // TODO remove when testDataMappingSmokeTest is enabled for Kudu
```

Comment:
Add this to previous commit (the one that removes `@Overwrite`)

Reply:
I add `@Deprecated` in `Add generic test for pushdown with nullable domain` commit because only after this commit the `testDataMappingSmokeTest` covers the use case being covered here